parlai/tasks/glue/agents.py

```diff
@@ -4,7 +4,22 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
+from parlai.core.teachers import MultiTaskTeacher
 from parlai.tasks.huggingface.agents import AbstractHuggingFaceTeacher
+from copy import deepcopy
+
+
+class AxTeacher(AbstractHuggingFaceTeacher):
+    """
+    Note: this is an evaluation dataset so it only has a test split
+    Use a model trained on MulitNLI to produce predictions for this dataset.
+    """
+
+    hf_path = 'glue'
+    hf_name = 'ax'
+    hf_text_fields = ['premise', 'hypothesis']
+    hf_label_field = 'label'
+    hf_splits_mapping = {'train': None, 'valid': None, 'test': 'test'}
```

Comment:
for evaluation datasets, changed mapping of train and valid to None e.g. `hf_splits_mapping = {'train': None, 'valid': None, 'test': 'test'}` and removed the task from the tests and the multitaskteacher, otherwise will produce errors. e.g. `display_data -t glue:ax` will work only if you set the datatype to test now.